src/examples/forms/tiles/TilesDefault.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Tiles } from '@zendeskgarden/react-forms';
+import { ReactComponent as LeafIcon } from '@zendeskgarden/svg-icons/src/16/leaf-stroke.svg';
+import { ReactComponent as ImageIcon } from '@zendeskgarden/svg-icons/src/16/file-image-stroke.svg';
+import { ReactComponent as PresentationIcon } from '@zendeskgarden/svg-icons/src/16/file-presentation-stroke.svg';
+
+const StyledCol = styled(Col)`
+  margin: ${p => p.theme.space.xs} 0;
+`;
+
+const Example = () => (
+  <Tiles name="example" aria-label="Tiles component example">
+    <Row>
+      <StyledCol xs={6} sm={4}>
```

Comment:
I think the overall example would be easier to rationalize (and nicer to design) if you just used `<Col sm={4}>` (no `xs` prop). In other words moving from side-by-side to stacked on mobile.

Reply:
Nice, ty. Much simpler.